examples/cython_polar/README.md

```diff
@@ -0,0 +1,53 @@
+# How to compile this example as a standalone executable
+
+Make sure to have done the instructions from the `README.md` file at the root of this repository before doing this.
+
+Copy the cmake configuration files from the AFF3CT build
+
+	$ mkdir cmake && mkdir cmake/Modules
+	$ cp ../../lib/aff3ct/build/lib/cmake/aff3ct-*/* cmake/Modules
+
+Compile the code on Linux/MacOS/MinGW:
+
+	$ mkdir build
+	$ cd build
+	$ cmake .. -G"Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-funroll-loops -march=native"
+	$ make
+
+Compile the code on Windows (Visual Studio project)
+
+	$ mkdir build
+	$ cd build
+	$ cmake .. -G"Visual Studio 15 2017 Win64" -DCMAKE_CXX_FLAGS="-D_SCL_SECURE_NO_WARNINGS /EHsc"
+	$ devenv /build Release my_project.sln
+
+The source code of this mini project is in `src/cython.cpp`.
+The compiled binary is in `build/bin/my_project`.
```

Comment:
If I'm right this is `build/bin/codec` instead of `build/bin/my_project`.

Reply:
Ack. Thanks.

Reply:
Done.